harvester/harvester
# Patch kubevirt image and bump LH to v1.4.2

**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

**Problem:**
- Need to cherry-pick the upstream kubevirt fix:
  - https://github.com/kubevirt/kubevirt/pull/8914
  - https://github.com/kubevirt/kubevirt/pull/9269
- Need to bump Longhorn to v1.4.2 to include some upstream fixes

**Solution:**
- Patch the kubevirt image from SUSE SLEs
- Bump Longhorn go mod and dependency chart to the v1.4.2 release

**Related Issue:**
- https://github.com/harvester/harvester/issues/3747

  **longhorn related:**
  - https://github.com/harvester/harvester/issues/3315
  - https://github.com/harvester/harvester/issues/3730
  - https://github.com/harvester/harvester/issues/2640
  - https://github.com/harvester/harvester/issues/3123
  - https://github.com/harvester/harvester/issues/1947
  
  **kubevirt related:**
  - https://github.com/harvester/harvester/issues/3586
  - https://github.com/harvester/harvester/issues/1289


**Test plan:**
- build a new ISO image based on the current PR
- verify all pods are running correctly on the `longhorn-system` namespace
- verify all the kubevirt pods are running correctly on the `harvester-system` namespace
- create a VM and check that LH-related features are working as expected. (e.g., snapshot, backup and volumes)
